tests/PHPUnit/Integration/CronArchiveTest.php

```diff
@@ -128,9 +128,10 @@ public function test_output()
 Total visits for today across archived websites: 1
 Archived today's reports for 1 websites
 Archived week/month/year for 1 websites
-Skipped 0 websites: no new visit since the last script execution
-Skipped 0 websites day archiving: existing daily reports are less than 150 seconds old
-Skipped 0 websites week/month/year archiving: existing periods reports are less than 3600 seconds old
+Skipped 0 websites
+- Skipped 0 websites: no new visit since the last script execution
+- Skipped 0 websites: existing daily reports are less than 150 seconds old
+- Skipped 0 websites: existing week/month/year periods reports are less than 3600 seconds old
```

Comment:
Cheers, done in https://github.com/piwik/piwik/commit/39e34edfc475a8a91a7c27a96682d7a06b3c5441